indico/modules/rb/client/js/components/TimeInformation.jsx

```diff
@@ -49,11 +49,25 @@ function TimeInformation({
             <Icon name="calendar outline" />
             {mEndDate && !mStartDate.isSame(mEndDate, 'day') ? (
               <Translate>
-                <Param name="startDate" wrapper={<strong />} value={mStartDate.format('L')} /> to{' '}
+                <Param
+                  name="startDateDay"
+                  wrapper={<span styleName="booking-times-weekday" />}
+                  value={mStartDate.format('ddd.')}
```

Comment:
```suggestion
                  value={mStartDate.format('ddd')}
```

this is included in `ddd` for languages that use it. currently you end up with this in french:

![image](https://user-images.githubusercontent.com/179599/234575360-70257db2-0217-4c31-bd02-23b66624bb79.png)